packages/frontend/component/src/components/overlay-modal/overlay-modal.tsx

```diff
@@ -0,0 +1,82 @@
+import { DialogTrigger } from '@radix-ui/react-dialog';
+import { memo, useCallback, useEffect } from 'react';
+
+import { Button, type ButtonProps } from '../../ui/button';
+import { Modal, type ModalProps } from '../../ui/modal';
+import * as styles from './overlay-modal.css';
+
+const defaultContentOptions: ModalProps['contentOptions'] = {
+  style: {
+    padding: 0,
+    overflow: 'hidden',
+    boxShadow: 'var(--affine-menu-shadow)',
+  },
+};
+const defaultOverlayOptions: ModalProps['overlayOptions'] = {
+  style: {
+    background: 'var(--affine-white-80)',
+    backdropFilter: 'blur(2px)',
```

Comment:
Seems unnecessary, because of blurring `#app` directly below in `useEffect`.

Reply:
In this ui component, useEffect seems to be unnecessary, because the background blur processed by useEffect does not change the background of the modal. I would use it alone for onboarding modal.

Reply:
<img width="1000" alt="image" src="https://github.com/toeverything/AFFiNE/assets/102217452/4a746cab-d89c-417c-8852-f1cf77e2450f">